include/net/aws_jobs.rst

```diff
@@ -1,35 +1,32 @@
 .. _lib_aws_jobs:
 
-AWS Jobs
+AWS jobs
 ########
 
-This library provides functions for working with the `AWS IoT jobs
-<https://docs.aws.amazon.com/iot/latest/developerguide/iot-jobs.html>`_
-service.
+The Amazon Web Services (AWS) jobs library provides functions for working with the `AWS IoT jobs`_ service.
 
-The AWS Jobs library provides APIs that can be used to:
-
-- report status,
-- subscribe to job topics.
+You can use the library to report the status of AWS IoT jobs and to subscribe to job topics.
 
 The module also contains the following elements:
 
-- string templates for topics,
-- defines for lengths of topics, status, and job IDs,
-- defines for Subscribe message IDs.
+- String templates for topics
```

Comment:
What do we want to say here? Is this what people should configure? Or is it just filler text? ;-)

Reply:
I think what we want to say is that this library contains a few string templates that can be used for generating MQTT topics(think of this as URLs where the device receives/sends messages) which are used for AWS Jobs.